test-network/scripts/deployCC.sh

```diff
@@ -33,51 +33,8 @@ CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:])
 
 FABRIC_CFG_PATH=$PWD/../config/
 
-# User has not provided a path, therefore the CC_NAME must
-# be the short name of a known chaincode sample
-if [ "$CC_SRC_PATH" = "NA" ]; then
```

Comment:
Let's see what @nikhil550 thinks... if we decide to remove all the chaincode defaults in favor of client passing them, then the defaults in lines 6-8 should be removed and we'd need to validate that client passed in the values.